modules/exploits/linux/http/nagios_xi_configwizards_authenticated_rce.rb

```diff
@@ -246,7 +262,7 @@ def execute_command(cmd, _opts = {})
       'uri' => '/nagiosxi/config/monitoringwizard.php',
       'cookie' => @auth_cookies,
       'vars_get' => url_params
-    }, 0)
+    })
```

Comment:
This was removed as I found it actually ended up preventing error detection in some cases. I'll add it back in once I finished testing but in one case it hid the NSP Dave error message as a result of it thinking the exploit had completed once the request had been sent, and then it didn't get a response immediately so it thought the payload must have failed to execute vs waiting around a bit.

Reply:
Got it. Does it need to be added back or was your experience that the exploit continued to work without it?

Reply:
Marking this as resolved for now since its not negatively affecting exploitation in my experience.